edgelet/contrib/config/linux/template.toml

```diff
@@ -0,0 +1,248 @@
+# source = "manual"
+# iothub_hostname = "example.azure-devices.net"
+# device_id = "my-device"
+# 
+# [provisioning.authentication]
+# method = "sas"
+#
+# device_id_pk = { value = "YXppb3QtaWRlbnRpdHktc2VydmljZXxhemlvdC1pZGU=" }     # inline key (base64), or...
+# device_id_pk = { uri = "file:///var/secrets/device-id.key" }                  # file URI, or...
+# device_id_pk = { uri = "pkcs11:slot-id=0;object=device%20id?pin-value=1234" } # PKCS#11 URI
+
+
+## Manual provisioning with X.509 certificate
+# [provisioning]
+# source = "manual"
+# iothub_hostname = "example.azure-devices.net"
+# device_id = "my-device"
+# 
+# [provisioning.authentication]
+# method = "x509"
+#
+# identity_cert = "file:///var/secrets/device-id.pem"                # file URI, or...
+# identity_cert = { method = "est", common_name = "my-device" }      # dynamically issued via EST, or...
+# identity_cert = { method = "local_ca", common_name = "my-device" } # dynamically issued by a local CA
+#
+# identity_pk = "file:///var/secrets/device-id.key.pem"                        # file URI, or...
+# identity_pk = { uri = "pkcs11:slot-id=0;object=device%20id?pin-value=1234" } # PKCS#11 URI
+
+
+## DPS provisioning with symmetric key
+# [provisioning]
+# source = "dps"
+# global_endpoint = "https://global.azure-devices-provisioning.net"
+# id_scope = "0ab1234C5D6"
+# 
+# [provisioning.attestation]
+# method = "symmetric_key"
+# registration_id = "my-device"
+#
+# symmetric_key = { value = "YXppb3QtaWRlbnRpdHktc2VydmljZXxhemlvdC1pZGVudGl0eS1zZXJ2aWNlfGF6aW90LWlkZW50aXR5LXNlcg==" } # inline key (base64), or...
+# symmetric_key = { uri = "file:///var/secrets/device-id.key" }                                                          # file URI, or...
+# symmetric_key = { uri = "pkcs11:slot-id=0;object=device%20id?pin-value=1234" }                                         # PKCS#11 URI
+
+
+## DPS provisioning with X.509 certificate
+# [provisioning]
+# source = "dps"
+# global_endpoint = "https://global.azure-devices-provisioning.net"
+# id_scope = "0ab1234C5D6"
+# 
+# [provisioning.attestation]
+# method = "x509"
+# registration_id = "my-device"
+#
+# identity_cert = "file:///var/secrets/device-id.pem"                # file URI, or...
+# identity_cert = { method = "est", common_name = "my-device" }      # dynamically issued via EST, or...
+# identity_cert = { method = "local_ca", common_name = "my-device" } # dynamically issued by a local CA
+#
+# identity_pk = "file:///var/secrets/device-id.key.pem"                        # file URI, or...
+# identity_pk = { uri = "pkcs11:slot-id=0;object=device%20id?pin-value=1234" } # PKCS#11 URI
+
+
+## DPS provisioning with TPM
+# [provisioning]
+# source = "dps"
+# global_endpoint = "https://global.azure-devices-provisioning.net"
+# id_scope = "0ab1234C5D6"
+# 
+# [provisioning.attestation]
+# method = "tpm"
+# registration_id = "my-device"
+
+
+# ==============================================================================
+# Cert issuance
```

Comment:
Asking for purposes of updating our product documentation on setting up a transparent gateway...  What's the mapping between what we have in the existing config.yaml and this new toml?

**Before**
certificates:
  device_ca_cert: "file:///..."
  device_ca_pk: "file:///..."
  trusted_ca_certs: "file:///..."

**After**??

Reply:
`trust_bundle_cert` near the top of the file plus the `[edge_ca]` section near the end.

Reply:
(See  edgelet/iotedge/test-files/config/ca-certs/super-config.toml )

Reply:
Got it, thanks!  These (trust bundle and device CA) were previously in closer proximity.  Recommend having them not appear as distant in the template file we provide so that they're easier for a customer to find when following docs to set up a transparent gateway.